exchanges/btcmarkets/btcmarkets_test.go

```diff
@@ -202,26 +202,48 @@ func TestGetTradeByID(t *testing.T) {
 	}
 }
 
-func TestNewOrder(t *testing.T) {
+func TestSubmitOrder(t *testing.T) {
```

Comment:
To be a huge jerk, I appreciate this test, however this does remove the test on just the API function. Could you please re-add the test and keep this one?